.eslintrc.json

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
```

Comment:
We have shared eslint config, doesn't it work here for some reason?

Reply:
@kozlovzxc-at-lido 
Didn't know about it and can't find it, where is it?

Reply:
Going to send it to DM, since it's still private

Reply:
Fixed